src/config/index.js

```diff
@@ -1,4 +1,33 @@
+require('dotenv').config();
+
+const { DB_DEV_NAME, DB_USER_NAME, DB_PASSWORD } = process.env;
+
```

Comment:
I understand that `dotenv` can be initialised only once and called anywhere within the app. So lines `1 & 3` are unnecessary. Thank you.

Reply:
could you elaborate more?

Reply:
That since `dotenv` has been imported and initialised `dotenv.config()` in `src/index.js`, there's no need to `import` and initialised in this file and any other file, you can just call the variables directly and it will work. That is, you already have access to `DB_DEV_NAME, DB_USER_NAME, DB_PASSWORD`.

Reply:
@tvpeter I can't get this to work, confirm this! perhaps i'm missing something. 
I've tried doing this with other projects I've worked on, what I found is you have to import it at least once like i did in config index.js then you can retrieve it anywhere else in your code from config.

Reply:
Oh yeah, you are right. your line `3` is needed but line `1` is not needed. So you have access to `process.env.DB_DEV_NAME`. But it is not necessarily needed in the `config/index` again since it's already initialised in `src/index.js`. Thank you.